### PR TITLE
fix(profiling): handle unsupported projects

### PR DESCRIPTION
**Pull request by eliashussary**
## Summary
Currently we don't render anything in scenarios where an org only has unsupported projects. 
In these situations we'll select the current/first unsupported project in their org and render the unsupported messaging.

To avoid any "minesweeper" style of UX, we've swapped out the dropdown menu w/ a grouped select control showing supported/unsupported projects. Unsupported projects are disabled. - This was previously used in our onboarding modal.

image
image
image

### CODE REVIEW

File: static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx [hunk 1]
```diff
@@ -102,39 +137,75 @@ export function ProfilingOnboardingSidebar(props: CommonSidebarProps) {
...
   }
 
+  if (!isSupported) {
+    // this content will only be presented if the org only has one project and its not supported
+    // in these scenarios we will auto-select the unsupported project and render this message
+    return (
+      <Fragment>
+        <div>
+          {tct(
+            'Fiddlesticks. Profiling isn’t available for your [platform] project yet but we’re definitely still working on it. Stay tuned.',
```
Comment (JonasBa) [inline static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx:196]: Can we say something like "reach out to profiling@sentry.io" or "ask on discord about platform support". I assume there may be a good part of platforms we wont support and dont want to give users the false impression that we are working on it

**Review by JonasBa — APPROVED** [review]

File: static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx (hunk 1 above)
Comment (eliashussary) [reply static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx:196]: Yeah good point, i'll fix it up

File: static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx [hunk 2]
```diff
@@ -128,20 +128,30 @@ export function ProfilingOnboardingSidebar(props: CommonSidebarProps) {
...
-            />
-          }
-          onChange={option => setCurrentProject(option.value)}
-          triggerProps={{'aria-label': currentProject?.slug}}
-          options={projectSelectOptions}
-          position="bottom-end"
-        />
+        <div
+          onClick={e => {
+            // we need to stop bubbling the CompactSelect click event
```
Comment (JonasBa) [inline static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx:133]: 😭

Comment (eliashussary) [reply static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx:133]: haha i fixed it in the sidebar but then my tests started failing and i was flipping tables, didn't want to start changing a bunch of unrelated code.

**Pull request merged**